mavis/python/mavis_tacplus_vault.py

```
@@ -0,0 +1,539 @@
+
+def _cache_put_user(username, password, groups):
+	"""Store user data in in-process cache."""
+	_user_cache[username] = {
+		"password": password,
+		"groups": groups,
+		"fetched_at": time.monotonic(),
+	}
+
+
+# Vault KV read ###############################################################
+def _vault_read_user(username):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `_vault_read_user`

(ANN202)